packages/remix-dev/cli/migrate/migrations/replace-remix-imports/index.ts

```diff
@@ -4,41 +4,116 @@ import { join } from "path";
 import type { PackageJson } from "type-fest";
 
 import * as jscodeshift from "../../jscodeshift";
-import { cleanupPackageJson } from "./cleanupPackageJson";
-import { getTransformOptions } from "./getTransformOptions";
+import {
+  findRemixDependencies,
+  getTransformOptions,
+} from "./getTransformOptions";
 import type { Options } from "./transform";
+import { isAdapter } from "./transform";
+import { isRuntime } from "./transform";
 import type { MigrationFunction } from "../../types";
 import { readConfig } from "../../../../config";
+import { hint } from "../../../../logging";
 
 const transformPath = join(__dirname, "transform");
 
+function* getTasks({
+  packageJson,
+  runtime,
+  adapter,
+}: {
+  packageJson: PackageJson;
+  runtime: string;
+  adapter?: string;
+}): Generator<string> {
+  let remixDeps = findRemixDependencies(packageJson.dependencies);
+
+  // runtime not in deps
+  if (!remixDeps.includes(runtime)) {
+    yield `Add \`@remix-run/${runtime}\` to your dependencies`;
```

Comment:
I think I'm leaning more towards "Install"/"Uninstall" (instead of "Add"/"Remove") to emphasize that we don't want to just delete a string from package.json, but we want to user to use their package manager to manipulate the dependencies across package.json, lock files, etc...

So I'm pretty sure I want to change this to "Install"/"Uninstall"

Reply:
Using install/uninstall instead of add/remove sounds indeed like a good idea to me 👍

Reply:
Using install/uninstall instead of add/remove sounds indeed like a good idea to me 👍